plugin-gestures/README.md

```diff
@@ -30,7 +30,7 @@ allprojects {
 
 // In the app build.gradle file
 dependencies {
-  implementation 'com.mapbox.plugin:maps-gestures:10.0.0-beta.19'
+  implementation 'com.mapbox.plugin:maps-gestures:10.0.0-beta.20'
```

Comment:
btw: gestures should also be dependent on `com.mapbox.plugin:maps-animation:10.0.0-beta.20`

Reply:
```suggestion
  implementation 'com.mapbox.plugin:maps-gestures:10.0.0-beta.20'
  // Mapbox Maps Gestures Plugin depends on the Mapbox Maps Animation Plugin
  implementation 'com.mapbox.plugin:maps-animation:10.0.0-beta.20'
```